src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -35,76 +35,58 @@
-            <number>0</number>
-           </property>
-           <item>
-            <widget class="QCheckBox" name="samplingCheckBox">
-             <property name="text">
-              <string>Enable callstack sampling</string>
-             </property>
-             <property name="checked">
-              <bool>true</bool>
-             </property>
-            </widget>
-           </item>
-           <item>
-            <layout class="QHBoxLayout" name="samplingPeriodMsHorizontalLayout">
-             <item>
-              <widget class="QLabel" name="samplingPeriodMsLabel">
-               <property name="text">
-                <string>Callstack sampling period (ms):</string>
-               </property>
-              </widget>
-             </item>
-             <item>
-              <widget class="QDoubleSpinBox" name="samplingPeriodMsDoubleSpinBox">
-               <property name="decimals">
-                <number>1</number>
-               </property>
-               <property name="minimum">
-                <double>0.100000000000000</double>
-               </property>
-               <property name="maximum">
-                <double>10.000000000000000</double>
-               </property>
-               <property name="singleStep">
-                <double>0.100000000000000</double>
-               </property>
-               <property name="value">
-                <double>1.000000000000000</double>
-               </property>
-              </widget>
-             </item>
-             <item>
-              <spacer name="samplingPeriodMsHorizontalSpacer">
-               <property name="orientation">
-                <enum>Qt::Horizontal</enum>
-               </property>
-               <property name="sizeHint" stdset="0">
-                <size>
-                 <width>40</width>
-                 <height>20</height>
-                </size>
-               </property>
-              </spacer>
-             </item>
-            </layout>
-           </item>
-          </layout>
+         <widget class="QCheckBox" name="samplingCheckBox">
+          <property name="text">
+           <string>Enable callstack sampling</string>
+          </property>
+          <property name="checked">
+           <bool>true</bool>
+          </property>
          </widget>
         </item>
+        <item>
+         <layout class="QHBoxLayout" name="samplingPeriodMsHorizontalLayout">
+          <item>
+           <widget class="QLabel" name="samplingPeriodMsLabel">
+            <property name="text">
+             <string>Callstack sampling period (ms):</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QDoubleSpinBox" name="samplingPeriodMsDoubleSpinBox">
+            <property name="decimals">
+             <number>1</number>
+            </property>
+            <property name="minimum">
+             <double>0.100000000000000</double>
```

Comment:
This seems a silly formatting, but I assume its coming from QtDesigner? Iff, then it should be left as is

Reply:
Indeed it's the Designer, so leaving as is.